lib/index.js

```diff
@@ -7,34 +7,88 @@ var APISync = function () {
   return {
     getAllWorkspaces: getAllWorkspaces,
     login: container.get('authenticationService').login,
+    cleanup: cleanup,
     pull: pull,
     setup: setup,
     status: status
   };
 
+  /**
+   * Returns all existing workspaces in the API Sync configuration.
+   *
+   * @return {Array} An array of all the configured workspaces.
+   */
   function getAllWorkspaces() {
     return container.get('workspaceRepository').getAll();
   }
 
-  function setup(directory, accessToken, strategy) {
-    setWorkingDirectory(directory);
-    setContext(accessToken, directory);
+  /**
+   * Cleanups the specified directory configuration.
```

Comment:
Fixed and added more details to the doc. The cleanup removes the configuration for the specified directory. This includes the selected Business Group and API Version.